.github/workflows/ci.yaml

```diff
@@ -48,6 +48,27 @@ jobs:
         run: |
           uv run pytest tests -m 'not integration'
 
+  langfuse-wrapper:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install UV
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
:nit: we've been switching `uses: astral-sh/setup-uv@v3` this way we lock versions

Reply:
going to just wait on this one until they're all updated here